src/slack/handlers/actions/log.ts

```diff
@@ -66,9 +66,8 @@ export const handleOpenLogModal: ActionMiddleware = async ({ body, ack, client }
 
 export const handleSubmitLogModal: ViewMiddleware = async ({ ack, body, view }) => {
     // Get the hours and task from the modal
-    let hours = safeParseFloat(view.state.values.hours.hours.value) ?? parseArgs(view.state.values.hours.hours.value ?? '').hours
+    let hours = (parseArgs(view.state.values.time.time.value ?? '').hours || safeParseFloat(view.state.values.time.time.value)) ?? NaN
     const activity = view.state.values.task.task.value
-
     // Ensure the time values are valid
     hours = isNaN(hours) ? 0 : hours
```

Comment:
Why don't you just include this with the definition of `hours`?

Reply:
This was kept in so that it could be used in the same way as the pre-rewrite modal, but with instant user feedback it makes more sense to remove this input option